packages/dashboard/.env.ropsten.example

```diff
@@ -0,0 +1,15 @@
+SIDETREE_METHOD='elem:ropsten'
+
+# Sidetree Variables
+MONGO_DB_CONNECTION_STRING=""
+DATABASE_NAME="element-ropsten"
+MAX_CONCURRENT_DOWNLOADS=20
+BATCH_INTERVAL_IN_SECONDS=5
+OBSERVING_INTERVAL_IN_SECONDS=5
+
+# Element Node Variables
+ELEMENT_CONTENT_ADDRESSABLE_STORE_SERVICE_URI=''
+ELEMENT_ANCHOR_CONTRACT=""
```

Comment:
So we want to add a default contract address here? `0x920b7DEeD5CdE055260cdDBD70C000Bbd5b30997`

Reply:
yes, for this file.

Reply:
Updated PR!